roles/openshift_node/tasks/main.yml

```diff
@@ -24,9 +21,7 @@
 # /etc/fstab and runs swapoff -a, if necessary.
 - name: Disable swap
   swapoff: {}
-
-- name: include node installer
-  import_tasks: install.yml
+  when: openshift_disable_swap | default(true) | bool
```

Comment:
We don't want the user to be able to disable swap.  https://github.com/openshift/openshift-ansible/pull/10605

Reply:
Oops, I copied that from 3.11. Removed